.github/actions/aws-sam-cli-develop/action.yml

```diff
@@ -0,0 +1,19 @@
+name: 'Clone and Build AWS SAM CLI'
+runs:
+  using: 'composite'
+  steps:
+    - uses: actions/checkout@v3
+      name: Checkout latest AWS SAM CLI commit
+      with:
+        repository: 'aws/aws-sam-cli'
+        ref: 'develop'
+        path: 'aws-sam-cli-develop'
+    - uses: actions/setup-python@v4
+      name: Install Python ${{ env.PYTHON_VERSION_INSTALL }}
+      with:
+        python-version: ${{ env.PYTHON_VERSION_INSTALL }}
+    - run: |
+        make -C aws-sam-cli-develop init
```

Comment:
Not sure I completely grok this line. Could you explain it to me?

Reply:
This `make` line just runs the make rule `init` in the `aws-sam-cli-develop` folder. The folder is manually specified since I didn't want to mess around with determining where the runner would clone the SAM CLI repo to, so I just cloned it into the working directory.